docs/integrations/index.rst

```diff
@@ -75,6 +69,26 @@ The `attrs` parameter adds a HTML attribute to the `<input>` tag that Bulma
 uses to add better styling. You may also add more Bulma-supported attributes
 to Polaris forms.
 
+Use an External Application for the Interactive Flow
+---------------------------------------------------
+
+.. _described in SEP-24: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#2-interactive-customer-information-needed
+
+Polaris provides `Form Integrations`_ for collecting and processing information
+about a deposit or withdraw. However if you would rather use another
+application to collect that information, override this function to return the
+URL that should be requested to begin that process.
+
+Note that if you choose to use another application or set of endpoints,
+you must make a ``postMessage`` call back to the window that opened the
```

Comment:
postMessage or the URL callback approach

Reply:
Instead of having the external application handle the callbacks which can be complicated, would it make sense to have them just redirect to a polaris `/interactive/complete` url which handles the callbacks properly?